server/matchmaker/search.py

```diff
@@ -128,17 +118,15 @@ def search_expansion(self) -> float:
     def match_threshold(self) -> float:
         """
         Defines the threshold for game quality
+        The base minimum quality is determined as 80% of the quality of a game
+        against a copy of yourself.
+        This is decreased by self.search_expansion if search is to be expanded.
 
         :return:
         """
-        thresholds = []
-        for rating in self.ratings:
-            _, deviation = rating
 
-            for d, q in self._deviation_quality.items():
-                if deviation >= d:
-                    thresholds.append(max(q - self.search_expansion, 0))
-        return min(thresholds)
+        quality_of_game_against_yourself = self.quality_with(self)
+        return max(0.8 * quality_of_game_against_yourself - self.search_expansion, 0)
```

Comment:
Keep in mind that a `Search` object may represent an entire party of players, and not necessarily just 1 player. We will need this to work for team matchmaking as well.

Reply:
I believe this should work as is but will add some tests.

Reply:
See `test_search_threshold_of_team_old_players_is_high`.